app/database/db.go

```diff
@@ -98,6 +105,10 @@ func (conn *db) Scan(dest interface{}) DB {
 	return &db{conn.DB.Scan(dest)}
 }
 
+func (conn *db) Count(dest interface{}) DB {
+	return &db{conn.DB.Count(dest)}
```

Comment:
would it be easier to use if returning `(int, DB)` instead? (open question)

Reply:
> would it be easier to use if returning `(int, DB)` instead? (open question)

I think it would not :( The method writes the data into the given argument similar to Scan(). I prefer keeping it similar to Gorm's method.